<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-HGCL64G" failures="1" tests="8" name="Sanity Test" time="63.935" errors="0" timestamp="2021-03-10T12:34:52 EST">
  <testcase classname="testNG.AssertionConcept" name="clickSignOnLink" time="0.304"/>
  <testcase classname="testNG.AssertionConcept" name="submitBtnEnable" time="0.044"/>
  <testcase classname="testNG.AssertionConcept" name="submitBtnEnableAfterCredentials" time="0.046">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at testNG.AssertionConcept.submitBtnEnableAfterCredentials(AssertionConcept.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- submitBtnEnableAfterCredentials -->
  <testcase classname="testNG.DataDrivenTestNGDataProvider" name="negativeSignInFunctionality" time="0.62"/>
  <testcase classname="testNG.DataDrivenTestNGDataProvider" name="negativeSignInFunctionality" time="0.565"/>
  <testcase classname="testNG.DataDrivenTestNGDataProvider" name="negativeSignInFunctionality" time="0.568"/>
  <testcase classname="testNG.DataDrivenTestNGDataProvider" name="positiveSignInFunctionality" time="0.599"/>
  <testcase classname="testNG.DataDrivenTestNGDataProvider" name="positiveSignInFunctionality" time="0.629"/>
  <testcase classname="testNG.AssertionConcept" name="clickSupportLink" time="0.0">
    <ignored/>
  </testcase> <!-- clickSupportLink -->
  <testcase classname="testNG.AssertionConcept" name="clickRegisterLink" time="0.0">
    <ignored/>
  </testcase> <!-- clickRegisterLink -->
</testsuite> <!-- Sanity Test -->
